addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -221,13 +289,15 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
                 $-pos: ($o-wsale-products-layout-grid-gutter-width / 2);
                 @include o-position-absolute($-pos, $-pos, $-pos, $-pos);
 
-                @if $o-wsale-products-layout-grid-gutter-width <= 0 {
-                    &.card {
-                        border: none;
+                &.card {
+                    border: none;
+
+                    @if $o-wsale-products-layout-grid-gutter-width > 0 {
+                        border-bottom: 1px solid lightgrey;
```

Comment:
I am scared of all your hardcoded colors, did you try with different color palettes / themes to see if it works? The grays can also be customized via the editor so I think this should really depend on those customized grays.